about.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link href="style.css" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+        <a class="log"><img src="img/quality logo.jpeg" alt="logo"></a> 
+    
+        <nav>
+          <ul class="header-list">
+            <li><a href="index.html" class="visited">Home</a></li>
+            <li><a href="#">sign up</a</li>
+            <li><a href="#">donate</a</li>
+            <li><a href="#">call us</a></li>   
+          </ul>
+        </nav>
+        <button class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </button>
+      </header>
+    <section class="section-head">
+      <div class="head-container">
+        <q class="head-subtitle">Hello, lovely people</q>
+        <h1 class="head-title">COMEDY OPEN MICS AT ACCRA NATIONAL THEATRE</h1>
+        <p class="head-text">
+            We are an independent show guide not a venue or show . We sell primary,
+            discount and resale tickets.
+        </p>
+      </div>
+    </section>
+    <section class="section-mark">
+      <h2 class="mark">Trademark</h2>
+      <div></div>
+      <p class="mark-desc">
+        This name and logo are registered trademarks of Elegant Interior Design
+      </p>
+      <div class="trademark-img">
+        <a class="log"><img src="img/quality logo.jpeg" alt="logo"></a> 
+      </div>
+    </section>
+    <section class="previous-work">
+      <h2 class="previous-title">Awards</h2>
+      <div class="previous-work-one"></div>
+    </section>
```

Comment:
- [ ] In the original version are two images with white text over them, please add an image and the text over them to be similar to the original design 👇🏽 

| Original design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/198769321-96236147-c6bd-4b66-9f69-c57d75c63790.png) | ![image](https://user-images.githubusercontent.com/58642949/198769504-59bed306-04b4-4c29-8f53-67353d810029.png) |